pkg/controller/login/reset_password.go

```diff
@@ -55,6 +55,11 @@ func (c *Controller) HandleSubmitResetPassword() http.Handler {
 			return
 		}
 
+		// Ensure that if we have a user, they have auth
+		if user, err := c.db.FindUserByEmail(form.Email); err == nil {
+			user.CreateFirebaseUser(ctx, c.client)
```

Comment:
can you add a metric for this?

realistically this should never happen (but we're seeing it)

Reply:
Metric added. I haven't used OpenCensus before - do these need to be declared or anything?